<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>619</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="documentMode">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>60</y>
      <width>70</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>Speed
Step Size</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_step">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>70</y>
      <width>70</width>
      <height>20</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>1/16</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>1/8</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>1/4</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>1/2</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>full</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="pushButton_inject">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>200</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Inject!</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_ports">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>30</y>
      <width>70</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>70</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Choose Port</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>240</y>
      <width>49</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Send text</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_retract">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>130</y>
      <width>70</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>retract</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_inject">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>110</y>
      <width>70</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>inject</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>110</y>
      <width>70</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>Mode
Select</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>160</y>
      <width>80</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Distance(mm)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_send">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>310</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Send</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_dataToSend">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>280</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_port_baudrate">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>30</y>
      <width>80</width>
      <height>20</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>9600</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>19200</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>38400</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>57600</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>115200</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="pushButton_connect">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>30</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Connect</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="plainTextEdit_log">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>60</y>
      <width>261</width>
      <height>191</height>
     </rect>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Shadow::Plain</enum>
    </property>
    <property name="lineWidth">
     <number>4</number>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_disconnect">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>30</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Disconnect</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_distance">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>160</y>
      <width>60</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_stop">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>200</y>
      <width>60</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Stop</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>619</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
